project_euler/problem_234.py

```diff
@@ -0,0 +1,31 @@
+# https://projecteuler.net/problem=234
+def fib(a, b, n):
+    temp = 0
+    if n==1:
+        return a
+    elif n==2:
+        return b
+    elif n==3:
+        return str(a)+str(b)
+    else:
```

Comment:
I think this else could be ignore. Because never will be go to else if return occur before. (Just a comment)

and temp=0 could be defined after the ```ifs```. This way we could avoid use memory when is not necessary.